dfs/volume.go

```diff
@@ -37,11 +37,15 @@ func (dfs *DistributedFilesystem) GetVolume(serviceID string) (*volume.Volume, e
 }
 
 // GetSubvolume gets the path of the *local* volume on the host
-func GetSubvolume(fsType, varpath, serviceID string) (*volume.Volume, error) {
+func GetSubvolume(fsType, varpath, serviceID string, mountVolume VolumeMounter) (volume.Volume, error) {
```

Comment:
Instead of passing in a function, we should instead create a fsType called "mockfs" or something, I think.